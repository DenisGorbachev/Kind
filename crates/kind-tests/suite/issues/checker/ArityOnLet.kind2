A (n : Data.U60) : Data.U60
A n = n

ArityOnLet : Data.U60
ArityOnLet = 
    let a = A
    2