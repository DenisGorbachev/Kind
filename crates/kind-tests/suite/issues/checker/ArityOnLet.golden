   ERROR  This function expects exactly 1 arguments but got 0

      /--[suite/issues/checker/ArityOnLet.kind2:6:13]
      |
    5 |    ArityOnLet = 
    6 |        let a = A
      |                v
      |                \This function requires a fixed number of arguments
    7 |        2


